packages/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -34,6 +34,7 @@ class GoogleMap extends StatefulWidget {
     this.polygons,
     this.polylines,
     this.circles,
+    this.padding,
```

Comment:
### **Question:** 
Though in the Android and iOS files if padding is null it defaults to 0, shouldn't this line be `this.padding = const EdgeInsets.all(0)` as it is used in `_GoogleMapOptions` by default just like other properties that are used in `_GoogleMapOptions`?